lib/Jsrt/JsrtHelper.cpp

```diff
@@ -89,7 +89,7 @@ void JsrtCallbackState::ObjectBeforeCallectCallbackWrapper(JsObjectBeforeCollect
         pthread_key_create(&s_threadLocalDummy, DISPOSE_CHAKRA_CORE_THREAD);
 #endif
 
-#if defined(CHAKRA_STATIC_LIBRARY)
+#if defined(CHAKRA_STATIC_LIBRARY) || !defined(_WIN32)
```

Comment:
For `ch` case I was talking about https://github.com/Microsoft/ChakraCore/blob/linux/bin/ch/ch.cpp#L659. I was thinking of removing that call and see if `ch` works.